sdk/Metrics/UpDownCounter.php

```diff
@@ -16,6 +32,7 @@ class UpDownCounter extends Counter implements API\UpDownCounter
     /**
      * Get $type
      *
+     * @access	public
      * @return  int
```

Comment:
Looks like these two docblocks don't line up?  Looks like one is spaced and the other is tabbed?